src/main/java/io/strimzi/kafka/bridge/BridgeContentType.java

```diff
@@ -16,4 +16,5 @@
     public static final String KAFKA_JSON_BINARY = "application/vnd.kafka.binary.v2+json";
     // JSON encoding
     public static final String KAFKA_JSON = "application/vnd.kafka.v2+json";
+    public static final String JSON = "application/json";
```

Comment:
`BridgeContentType` is supposedly about "bridge supported content types" (according to its Javadoc), but this content type cannot be used as the content type for a message, so I don't think this content type really belongs here. Alternatively if `BridgeContentType` is really just some constants for content types then it should be renamed to `ContentType`.

Reply:
From the beginning, I decided to use `Bridge` prefix because these content types were strictly related to the contents that the bridge is able to handle. Actually, clients should be able to handle `application/json` as well together with `application/vnd.kafka.json.v2+json` and `application/vnd.kafka.v2+json`. So actually this is a bridge supported content type.

Reply:
Fair enough.